src/builder/cleanup.js

```diff
@@ -0,0 +1,10 @@
+import { join } from "path"
+import { unlinkSync } from "fs"
+
+export default (wd, fileToKeep, assets) => {
+  assets
+    .filter((file) => file !== fileToKeep)
+    .forEach((file) => {
+      unlinkSync(join(wd, file))
```

Comment:
How about using `unlink` promise and `Promise.all` ? I think it would help the perf